.eslintrc.json

```diff
@@ -24,6 +24,7 @@
     "no-underscore-dangle": 0,
     "no-console": 0,
     "no-unused-vars": 0,
+    "no-constant-condition": 0,
```

Comment:
just look at the diff, if we dont want to fix linter issues why have it at all. It would be easier to just remove config and let those interested in linting have their own config.